src/sprotty/theia-diagram-server.ts

```diff
@@ -59,6 +74,18 @@ export class TheiaDiagramServer extends DiagramServer {
             this.resolveConnector = resolve)
     }
 
+    getConnector() {
```

Comment:
Just curious, why did you choose Java-style getters here instead of TypeScript getters?

Reply:
Because it's a promise.